src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/endpoint/CaseEndpointCaseLaunchTest.java

```diff
@@ -69,19 +69,34 @@ public void getCaseById_GoodId() throws Exception {
 
   @Test
   public void getCaseById_BadId() throws Exception {
-    ResultActions actions = mockMvc.perform(getJson("/cases/123456789/launch?agentId=12345"));
+    ResultActions actions =
+        mockMvc.perform(getJson("/cases/123456789/launch?agentId=12345&individual=true"));
     actions.andExpect(status().isBadRequest());
   }
 
   @Test
   public void getCaseById_GoodIdMissingAgentTest() throws Exception {
-    ResultActions actions = mockMvc.perform(getJson("/cases/" + uuid + "/launch"));
+    ResultActions actions = mockMvc.perform(getJson("/cases/" + uuid + "/launch?individual=true"));
+    actions.andExpect(status().isBadRequest());
+  }
+
+  @Test
+  public void getCaseById_GoodIdBadAgent() throws Exception {
+    ResultActions actions =
+        mockMvc.perform(getJson("/cases/" + uuid + "/launch?agentId=ABC45&individual=true"));
     actions.andExpect(status().isBadRequest());
```

Comment:
The test methods throughout this class need to be renamed to reflect that they are testing eq launch and not get case

Reply:
Agree. Method names updated.